docs/development/triage-guide.md

```diff
@@ -0,0 +1,89 @@
+# Triage guide
+
+## What is triage?
+
+Triage is basically filtering the issues/discussions, and categorizing them with the proper labels.
+
+## What a triagist is allowed to do
+
+If you've been given triage rights, you are allowed to do the following things:
+
+- Apply labels to issues
+- Close, reopen, and assign all issues and pull requests
+- Mark duplicate issues and pull requests
+- Request pull request reviews
+- Lock and unlock discussions
+- Individually convert issues to discussions
+
+**Note:** We don't use milestones or project boards.
+
+## Guidelines for triage workflow
+
+The following are guidelines as we cannot cover all situations.
+Use common sense, and just do your best, and you'll do all right.
+Don't be afraid to ask for help.
+
+### Apply labels to issues
+
+All issues should have labels attached to them.
+Read the [issue-labeling guide](https://github.com/renovatebot/renovate/blob/master/docs/development/issue-labeling.md) to get all the necessary info.
+
+In general try to make a good-faith effort to label things correctly.
+
+### Closing issues
+
+You can close a issue yourself if it's:
+
+- Spam
+- Obviously fixed
+
+For really old issues, it's probably a good idea to ask the maintainers to decide if they want to keep or close the issue.
+
+### Closing pull requests
+
+It's not very often that you'll need to close a PR, but you can certainly do it in case of spam or malicious content in the PR diff.
+
+### Reopen issues
+
+Sometimes a bug is fixed with a PR that links to a issue.
+When the PR is merged, the issue is automatically closed.
+Sometimes the bug was not really fixed, and someone says: "Hey this is still broken for me."
+In that case, re-open the issue.
+
+### Assign issues
+
+You can assign a issue to yourself, so that others know you're going to work on the issue.
+You can also assign another user, if they have said they want to volunteer to fix the bug or implement a feature.
+
+### Mark duplicate issues and pull requests
+
+If you see an issue that's an obvious duplicate, you can mark it as a duplicate and close it.
```

Comment:
Should we have a `duplicate` label for this? I haven't seen GitHub's native functionality working.

Reply:
Let's see what the [GitHub docs, about duplicate issues and pull requests](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-duplicate-issues-and-pull-requests) has to say about this:

> For a "marked as duplicate" timeline event to appear, the user who creates the duplicate reference comment **must have write access to the repository** where they create the comment.

> To mark an issue or pull request as a duplicate, type "Duplicate of" followed by the issue or pull request number it duplicates in the body of a new comment. 

So I as a triagist cannot use this anyways, as I don't have rights to mark anything as a duplicate.
I saw that **viceice** was able to mark a issue as a duplicate (he has at least write level rights).
You should also be able to use "Duplicate of" functionality as well (as your an admin, and so can do as you want...).

A `duplicate` label does not really add anything, I think?
Might just as well ping somebody who does have write rights to mark it as a "proper duplicate of issue/pr"?

Maybe the text should be:

```suggestion
If you see an issue that's an obvious duplicate, ask for somebody with write rights to use the "Duplicate of" functionality.
For more on the "Duplicate of" functionality read the [GitHub docs, about duplicate issues and pull requests](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-duplicate-issues-and-pull-requests).
```

Reply:
I was thinking it could be useful in future for counting bugs opened and closed, features added, etc. So then it's nice to be able to de-duplicate. I'm leaning towards using a label so that such queries would be simple too.